lib/src/services/gamepad.dart

```diff
@@ -0,0 +1,42 @@
+import "package:win32_gamepad/win32_gamepad.dart";
+
+import "service.dart";
+
+///A service to send the comtroller's activity
+///using win32_gamepad
```

Comment:
See if you can expand on this. See the documentation on [`UdpClient`](https://binghamtonrover.github.io/Dashboard/services/UdpClient-class.html) for an example (no need for _that_ much, just get the gist of it). 

Generally, documentation for a class should look like this:
```dart
/// This is a sentence that says what the class is/does.  
///
/// Here's a little background on what the class is doing, and a basic rundown of how it's doing it. 
/// Use multiple sentences and lines to get your point across, but stick to one paragraph.
///
/// Here's some info on how to use the class, now that we know what it does and some terminology. This
/// should be a paragraph, bullet, list, code sample, etc.
///
/// Finally, if the class is _really_ interesting, put more details here. Try to only involve the overall details that
/// are relevant to the class as a whole, not something that can be documented on a method instead.
class GamepadService extends Service { 
```